docs/source/guide/lre-3-options.md

```diff
@@ -0,0 +1,282 @@
+
+
+```python
+lre_value = mitiq.lre.execute_with_lre(
+    circuit,
+    executor,
+    degree,
+    fold_multiplier,
+    folding_method=<"noise scaling method imported from zne.scaling.folding">,
+    num_chunks=<"number of chunks to group a large circuit into">,
+)
+```
+
+The hyperparameters that can be controlled are:
+
+
+- `degree`: to modify the extrapolating polynomial
+- `fold_multiplier`: to control how the circuit is scaled
+- `folding_method`: to choose the unitary folding method
+- `num_chunks`: to alter the sampling cost
+
+
+## Controlling how the noise-scaled circuits are created
+
+
+### Extrapolating polynomial
+
+
+Depending on the choice of the degree of the polynomial, how each noise-scaled circuit is scaled varies. This
+in turn influences how the sample matrix for multivariate Richardson extrapolation is created.
+
+
+Suppose we have an input circuit consisting of 4 layers.
+
+
+```{code-cell} ipython3
+from cirq import LineQubit, Circuit, CZ, CNOT, H
+
+
+q0, q1, q2, q3  = LineQubit.range(4)
+circuit = Circuit(
+   H(q0),
+   CNOT.on(q0, q1),
+   CZ.on(q1, q2),
+   CNOT.on(q2, q3),
+)
+
+
+print(circuit)
+```
+
+
+How the circuits are scaled is controlled by `degree`, `fold_multiplier`, and the number of layers in the input circuit. We choose `degree = 2` and `fold_multiplier = 2` for demonstration purposes.
+
+
+
+For `degree = 2`, the terms in the monomial basis are given below where the subscripts correspond to the 4 layers in the
+circuit.
+
+
+$$\{1, λ_1, λ_2, λ_3, λ_4, λ_1^2, λ_1 λ_2, λ_1 λ_3, λ_1 λ_4, λ_2^2, λ_2 λ_3, λ_2 λ_4, λ_3^2, λ_3 λ_4, λ_4^2\}$$
+
+
+```{seealso}
+These terms in the monomial basis are also used to define the rows of the sample matrix used for extrapolation.
+```
+
+
+```{code-cell} ipython3
+from mitiq.lre.multivariate_scaling import get_scale_factor_vectors
+
+scale_factors = get_scale_factor_vectors(circuit, degree=2, fold_multiplier=2)
+
+scale_factors
+```
+
+
+In the noise scaled circuits created using the above scale factor vectors:
+
+- The term $1$ in the monomial terms basis corresponds to the `degree = 0` term in the polynomial which is equivalent to
+the $\lambda_1^0\lambda_2^0\lambda_3^0\lambda_4^0$ term. Due to this term, the first noise-scaled circuit is unchanged.
+
+- due to the $λ_1$ term in the monomial basis, the second noise-scaled circuit only scales the first layer in the circuit.
+
+- due to the $λ_2$ term in the monomial basis, the next noise-scaled circuit only scales the second layer in the circuit
+
+- and so on.
+
+```{code-cell} ipython3
+print(f"Total number of noise scaled circuits created: {len(scale_factors)}")
+```
+
+As the `fold_multiplier` is changed, the number of scaled circuits remains the same but how the layers are scaled
+is altered.
+
+```{code-cell} ipython3
+
+
+scale_factors_diff_fold_multiplier = get_scale_factor_vectors(
+   circuit,
+   degree=2,
+   fold_multiplier=3)
```

Comment:
```suggestion
scale_factors_diff_fold_multiplier = get_scale_factor_vectors(
    circuit,
    degree=2,
    fold_multiplier=3,
)
```

There are a few other places in this PR where 3 spaces are used for indentation. Can you change them to 4?